srv/pom.xml

```diff
@@ -192,7 +192,7 @@
 							<goal>npm</goal>
 						</goals>
 						<configuration>
-							<arguments>install @sap/cds-dk@${cds.install-cdsdk.version} @sap/cds-mtxs@^2 --no-save</arguments>
+							<arguments>install @sap/cds-dk@${cds.install-cdsdk.version} @sap/cds-mtxs@^2 --ignore-scripts --no-save</arguments>
```

Comment:
Is this required to skip the `postInstall` script of the hdi-deploy? I assume it will skip also other scripts which may be required. Do you see any side-effects ?

Reply:
Yes, exactly. I haven´t seen any side effects of `--ignore-scripts`. The alternative solution would be to install `@cap-js/hana`. This will install the necessary hana db client.

Reply:
ok, then let's use this approach for now. Thanks for clarification.